src/pages/index.tsx

```diff
@@ -51,6 +52,9 @@ const modeDispatchPages: Record<AppMode, () => React.ReactElement> = {
           <Route path="/leaderboard">
             <LeaderboardPage />
           </Route>
```

Comment:
@AbdulWahab3181 please pull from master there is a route and component for this already ```sh          
<Route path="/feature/:feature_uuid">
     <WorkspaceFeature />
 </Route>
```